.travis.yml

```diff
@@ -0,0 +1,32 @@
+sudo: false
+
+language: python
+
+python:
+  - 2.7
+  - 3.2
+  - 3.3
+  - 3.4
+  - "pypy"
+  - "pypy3"
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+
+addons:
+  apt:
+    sources:
+      - build-essential
+
+install:
+ - pip install --upgrade pip
+ - pip install --install-option="--no-cython-compile" Cython 
+ - pip install .
+
+script:
+ - cd `mktemp -d`
+ - nosetests -v nxlemon
```

Comment:
I am not particular sure about that. In the case of `networkx`, it could be doing a file system search instead of Python module search. But I may be wrong. For now I will assume that it works. We will know if that is the case when you set up the first test case.